templates/minimal/README.md

```diff
@@ -1,13 +1,93 @@
-# Minimal Template
+<div align="center">
 
-This is a minimal template for creating a blockchain using the Polkadot SDK.
+# Polkadot SDK's Minimal Template
 
-# Docs
+<img height="70px" alt="Polkadot SDK Logo" src="https://github.com/paritytech/polkadot-sdk/raw/master/docs/images/Polkadot_Logo_Horizontal_Pink_White.png#gh-dark-mode-only"/>
+<img height="70px" alt="Polkadot SDK Logo" src="https://github.com/paritytech/polkadot-sdk/raw/master/docs/images/Polkadot_Logo_Horizontal_Pink_Black.png#gh-light-mode-only"/>
 
-You can generate and view the [Rust
-Docs](https://doc.rust-lang.org/cargo/commands/cargo-doc.html) for this template
-with this command:
+> This is a minimal template for creating a [Substrate](https://substrate.io/) blockchain.
+> 
+> This template is automatically updated after releases in the main [Polkadot SDK monorepo](https://github.com/paritytech/polkadot-sdk).
+
+</div>
+
+## Getting Started
+
+🛠️ Depending on your operating system and Rust version, there might be additional
+packages required to compile this template.
+
+👉 Check the
+[Substrate Install](https://docs.substrate.io/install/) instructions for your platform for
```

Comment:
For this, we have: 

rust-setup.md in solochain template
and `https://getsubstrate.io/` is a legacy project that needs to be archived.

For now I suggest is to remove both of these, add one single bash file to `polkadot-sdk` that installs latest rust and whatever else you need and reference to it.

Reply:
There used to be [this `init.sh` script](https://github.com/paritytech/polkadot/blob/master/scripts/init.sh) in the old repo that did this.

Do you know why it was abandoned and eventually removed?

Reply:
For now I wrote a minimal instruction, until we have a script or a doc that could be linked.